pages/glossary/section-of-repeated-content.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Section of repeated
+key: section-of-repeated-content
+---
+
+A [section of content][] of an [HTML web page][html web page] is a _section of repeated content_ if there exists [sections of content][section of content] serving the same purpose or containing the same key content in other [HTML web pages][html web page] in the same [set of web pages](https://www.w3.org/TR/WCAG21/#dfn-set-of-web-pages).
+
+The content does not need to be exactly identical to be repeated content. For example, an advertisement sidebar appearing on all pages of a website is a section of repeated content even if each page displays a different advertisement, the common purpose being advertisement.
```

Comment:
This makes it so that breadcrumbs are repeated content (because it serves the same purpose), I don't think we want to not allow skipping to the breadcrumbs with a skiplink.

Reply:
I do agree that breadcrumbs are "sections of repeated content" according to that definition.
(and in any case, they are "sections of content")

I am less sure on whether they need skiplink or similar mechanisms the same way the other sections do. I can't find anything on this topic…

---
I do agree that requiring a heading at the start of the breadcrumbs seem silly (although the "you are here" text could very well have a `role="heading"` if needed…) But at the same time the breadcrumb do create several tab-stop and a skiplink might be a good idea (especially if there's a long list of breadcrumbs, which is the case where they are most useful…)

---
I'm also not sure how to change the definition (or rules) to exclude breadcrumbs (adding an explicit exception seems bad).

What would be the defining characteristic(s) of breadcrumbs that make them not require a skiplink/header/direct link from the top/toggle visibility button?

---
https://www.smashingmagazine.com/2009/03/breadcrumbs-in-web-design-examples-and-best-practices/#5-breadcrumbs-with-sub-navigation has some example of breadcrumbs with sub navigation that would probably need a skiplink because they're actually huge (this is a pretty old page (2009) and current trend in breadcrumb seems to be different).

https://codepen.io/accessabilly/pen/mRyjOm (from https://accessabilly.com/breadcrumb-trails-come-in-and-find-out/) is using heading for the "you are here" text of the breadcrumb. But that seem to be the exception from what I'm seeing (and https://www.w3.org/TR/wai-aria-practices-1.1/examples/breadcrumb/index.html doesn't do that).

---
On a somewhat related topic, WCAG considers search boxes as full-fledged sections of content for the bypass block purpose. Notably, G124 (set of links to sections of content) has an example where there is a link to the searchbox; and H69 (headings) has an example with heading for the search field.

So it looks like WCAG is considering search field as requiring the same mechanism as other sections of content. Would you fail a page that is missing a skiplink, (or a heading, or a toggle visibility button), on a searchbox?

I fail to see the fundamental difference between breadcrumbs and searchboxes that would require heading/direct links/… for one and not for the other.

---
After talking with @kasperisager, we kinda agree that skiplinks/headings/direct links/toggle button are only one way to pass "Bypass blocks", and that, for example, if one choose to use direct links to sections of content at the top of the page, then one has to do it fully and there is need to have links also to breadcrumbs, searchbox or ads… But alternatively, one could just have a link to main content and be done.

---
What is your opinion on all that?
What would be the defining characteristic(s) of breadcrumbs making them different from searchboxes who need headings (according to H69), direct links from the top (according to G124), or skiplinks/toggle buttons (extrapolating previous examples)?

Reply:
Breadcrumbs are uniquely tailored to the page within the set of web pages. It is by definition a unique piece of content for that page. Search boxes are the same on every page.

I always passed websites that have skiplinks that link to the breadcrumb because it is the first unique (non-repeating content) on the page.

Advertisement is not unique to the page and there is a chance of getting the same advertisement twice, definitely making it repeating content.

I think the definition can simply drop "serving the same purpose". It would still include search boxes and advertisement, but would not include breadcrumbs.